dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-body.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<title>non-passive mousewheel event listener on body</title>
+<link rel="help" href="https://github.com/WICG/interventions/issues/64">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-actions.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script src="resources/wait-for.js"></script>
+<script src="resources/scrolling.js"></script>
+<script>
+  onload = () => {
```

Comment:
I thought if you called promise_test during onload you needed to use explicit_done? I guess I am probably just wrong, the wpt.fyi runs show that the tests are fine, so feel free to resolve... I am just a bit confused.

Is there a particular reason to delay for onload, though? Since everything is a sync script it should be basically equivalent... maybe if you move the `<div>` before, it would be unnecessary.

Reply:
That sounds familiar. Maybe something changed in testharness.js? I can change the test to avoid onload though.